CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+<!-- For now we follow Gutenberg's package changelog format https://github.com/WordPress/gutenberg/tree/HEAD/packages#maintaining-changelogs. -->
+
+## 1.1.0 (2021-05-19)
+
+### Breaking Changes
+
+- Removed `...AbstractChainedJob::filter_items_before_processing` method ([#2](https://github.com/woocommerce/action-scheduler-job-framework/pull/2))
+
+### New Feature
+- Added `...AbstractChainedJob::process_items` method ([#2](https://github.com/woocommerce/action-scheduler-job-framework/pull/2))
```

Comment:
Nice to use markdown format! Issue/PR links help clarify changes, can dig back through to get the context or history.

With these changes specifically, the changelog items could focus more on why - what the change enables the library client to do.

Reply:
I took a stab after seeing the change in context on https://github.com/woocommerce/facebook-for-woocommerce/pull/1924#discussion_r634974904.

> Added `AbstractChainedJob::process_items()` method. Clients can override to use state across the batch job.

(I'm finding this tricky to explain!)

Reply:
Sounds good, I added some more context in 48c94e2e01ebe9acff8be687c1aecfb8883d98af